### ISSUE TITLE
Helm Chart support for providing imagePullSecrets to pull from private image repositories.

### ISSUE THREAD

**Issue by jbmtalbot** [rdvansloten/cert-manager-key-vault-sync#8]
Great project. It does one thing very well and is exactly what we needed to push certificates to Azure Key Vault for easy pulling with external-secrets. 

For those of us required to pulling images from private repositories only, support for imagePullSecrets in the Helm Chart would be helpful. It could be implemented either in the ServiceAccount or Deployment and the user supplies the imagePullSecret as a Kubernetes secret in the same namespace.

Example 
```yaml
---
apiVersion: v1
kind: ServiceAccount
imagePullSecrets:
- name: <kubernetes-secret>
...
```

```yaml
---
apiVersion: apps/v1
kind: Deployment
...
spec:
  template:
    spec:
      imagePullSecrets:
      - name: <kubernetes-secret>
...
```

**Comment by rdvansloten** [comment]
Great suggestion! Seems easy enough to implement. Will probably push it out somewhere this week 🙂

**Comment by rdvansloten** [comment]
@jbmtalbot Change has been implemented in Helm Chart version v0.2.2!

See `helm pull oci://registry-1.docker.io/rdvansloten/cert-manager-key-vault-sync --version v0.2.2`
